nncf/torch/nncf_network.py

```diff
@@ -31,15 +29,14 @@
 from nncf.common.graph import MODEL_OUTPUT_OP_NAME
 from nncf.common.graph import NNCFGraph
 from nncf.common.graph import NNCFNode
-from nncf.common.graph import NNCFNodeExpression
 from nncf.common.graph import NNCFNodeName
-from nncf.common.graph.graph_matching import NodeExpression
 from nncf.common.graph.model_transformer import ModelTransformer
 from nncf.common.graph.transformations.commands import TargetType
 from nncf.common.graph.transformations.commands import TransformationPriority
-from nncf.common.hardware.config import HWConfig
+from nncf.torch.graph.patterns import GraphPattern
```

Comment:
The pattern matching should be made common since it will only operate on the nx graph underlying the common NNCFGraph representation.

Reply:
Done